test/test_pairs_classifiers.py

```diff
@@ -73,7 +73,7 @@ def test_raise_not_fitted_error_if_not_fitted(estimator, build_dataset,
   estimator.set_params(preprocessor=preprocessor)
   set_random_state(estimator)
   with pytest.raises(NotFittedError):
-    estimator.predict(input_data)
+    estimator.decision_function(input_data)
```

Comment:
a few things here:
- description of the test still mentions "predict"
- maybe all relevant methods should be tested here? it would have detected that there was a problem for `score_pairs`

Reply:
Addressed in PR #269, sorry for the mishaps.